.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "liveServer.settings.port": 5503
+}
```

Comment:
[OPTIONAL] since this is your own VSCode setting. it doesn't necessarily need to be committed into the repo. so having this in the `.gitignore` file may be a good choice 👍